pkgs/applications/video/mplayer/default.nix

```diff
@@ -218,6 +218,6 @@ stdenv.mkDerivation rec {
     homepage = http://mplayerhq.hu;
     license = "GPL";
     maintainers = [ stdenv.lib.maintainers.eelco ];
-    platforms = stdenv.lib.platforms.linux ++ stdenv.lib.platforms.darwin;
+    platforms = [ "i686-linux" "x86_64-linux" "x86_64-darwin" ];
```

Comment:
whats the reason for this platform change?

Reply:
I went back as far as the hydra history goes (2.5y), https://hydra.nixos.org/job/nixpkgs/trunk/mplayer.aarch64-linux/all?page=14 and aarch64 didn't ever build.